### ISSUE TITLE
Unable to create context for MSSQL connection -- DBAPIError: (pyodbc.Error) ('07002')

### ISSUE THREAD

**Issue by ghost** [fivetran/great_expectations#3027]
**Describe the bug**
After successfully setting up the sqlalchemy connection for MSSQL, creating the batch context for the expectation suite fails.

**To Reproduce**
1. run `great_expectations datasource new `
2. setup mssql datasource with the help of sqlalchemy connection string
3. run `great_expectations suite new`
4. run the first cell creating cotext and defining batch data
5. See error below

```
DBAPIError: (pyodbc.Error) ('07002', '[07002] [Microsoft][ODBC Driver 17 for SQL Server]COUNT field incorrect or syntax error (0) (SQLExecDirectW)')
[SQL: SELECT TOP ? * 
into #ge_tmp_b847a7a1 from dbo.fzs]
(Background on this error at: http://sqlalche.me/e/14/dbapi)
```

**Expected behavior**
context and batch successfully created and returns the first couple of rows

**Environment (please complete the following information):**
 - Operating System: Windows
 - Great Expectations Version: 0.13.21

**Additional context**
None

**Comment by talagluck** [comment]
Hi @ghost - thanks for raising this, and apologies for delays! This may have been resolved for V3 by #3353. Would you please confirm whether this is an issue when running V3?

**Comment by talagluck** [comment]
I'm going to close this issue for now, but feel free to re-open if you find that the issue persists for you.
